proxy/installer/configure-proxy.sh

```diff
@@ -325,9 +325,12 @@ SQUID_DIR=/etc/squid
 UP2DATE_FILE=$SYSCONFIG_DIR/up2date
 SYSTEMID_PATH=$(awk -F '=[[:space:]]*' '/^[[:space:]]*systemIdPath[[:space:]]*=/ {print $2}' $UP2DATE_FILE)
 
-systemctl status salt-minion > /dev/null 2>&1 || systemctl status venv-salt-minion > /dev/null 2>&1
-if [ $? -eq 0 ]; then
-    /usr/sbin/fetch-certificate $SYSTEMID_PATH
+PYTHON_ENV=""
+systemctl is-active --quiet salt-minion && PYTHON_ENV="/usr/bin/python -u "
```

Comment:
`/usr/bin/python` is Python2, is that on purpose?

Reply:
It must be running with `python3` for sure

Reply:
is it possible it's the cause of the issue https://github.com/uyuni-project/uyuni/issues/5573#issuecomment-1187347996 ? Maybe `fetch-certificate.py` would work just using `#!/usr/bin/python3 -u` as shebang and adding `/usr/lib/venv-salt-minion/bin/python` is not required

Reply:
I think no, before fetch-certificate was executed with the python specified in the shebang and it will just fail with python2 as we are not shipping salt module for it.